spec/01-unit/011-balancer_spec.lua

```diff
@@ -562,6 +583,27 @@ describe("Balancer", function()
         })
         assert.are.same(crc32(table.concat(value)), hash)
       end)
+      describe("cookie", function()
```

Comment:
It looks like this block is the same as the above, it should exercise `"cookie"` as the `hash_fallback` entry.

Reply:
Woops! Fixed